src/.editorconfig

```diff
@@ -0,0 +1,242 @@
+# editorconfig.org
```

Comment:
Since this config is down one level in the folder hierarchy, it will overwrite the rules in the SDL editorconfig, allowing us to merge the two rulesets together

Reply:
We should have 1 .editorconfig for all code.